### DISCUSSION TITLE
Left padding.

### DISCUSSION THREAD

**Discussion by dharmx** [hrsh7th/nvim-cmp#929] [Q&A] [answered]
Is there any way to discard the padding on the left side of the completion menu?

![image](https://user-images.githubusercontent.com/80379926/164845991-02be84c8-e243-4331-bfeb-d556cb964b37.png)

**Comment by Shougo** [comment]
Well, what do you means the padding?

**Reply by dharmx** [reply]
![image](https://user-images.githubusercontent.com/80379926/164877967-7bf28167-d19f-405a-ad5d-8a22d5ef3a4f.png)

This gray area on the left

**Reply by Shougo** [reply]
Thanks.  I get it.

**Comment by gegoune** [comment, marked as answer]
Duplicate https://github.com/hrsh7th/nvim-cmp/issues/846

**Reply by dharmx** [reply]
Aww...
